Server/VikingVault.Services/Exceptions/BankAccountServiceException.cs

```diff
@@ -6,6 +6,5 @@ namespace VikingVault.Services.Exceptions
 {
     class BankAccountServiceException:Exception
```

Comment:
Please make it public and, like for TransactionException add a constructor with a string as a parameter. And format the code to have spaces before and after ":"